third-party/rsutils/src/hexarray.cpp

```diff
@@ -0,0 +1,90 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#include <rsutils/string/hexarray.h>
+#include <rsutils/string/from.h>
+#include <rsutils/string/slice.h>
+#include <rsutils/string/hexdump.h>
+
+#include <nlohmann/json.hpp>
+
+
+namespace rsutils {
+namespace string {
+
+
+/*static*/ std::string hexarray::to_string( bytes const & bytearray )
```

Comment:
what about this "static" ? should be commented out or not ?

Reply:
I put these there on purpose, since it's not always clear that you're looking at a static function member body. I find it helps me understand where I am.